### ISSUE TITLE
Demo on webcomponents.org not visible

### ISSUE THREAD

**Issue by ghost** [vpusher/paper-tree#11]
I can't see the demo on https://www.webcomponents.org/element/vpusher/paper-tree/elements/paper-tree. It used to be there but disappeared after the last updates.

**Comment by vpusher** [comment]
Yep, you are right. We probably need to set a value to `@demo` for now.
